docs/docs/Deployment/deployment-kubernetes.md

```diff
@@ -312,59 +287,69 @@ env:
       secretKeyRef:
         name: openai-key
         key: openai-key
-
 ```
 
+or directly from the values file (not recommended for secret values):
 
-or directly from the values file (not recommended for secret values!):
-
-
-```yaml
+```
 env:
   - name: openai_key_var
     value: "sk-...."
+```
 
+Alternatively, add a Kubernetes secret with Helm commands.
+
+1. Create the secret:
+```shell
+kubectl create secret generic openai-credentials \
+  --namespace langflow \
+  --from-literal=OPENAI_API_KEY=your-api-key-here
 ```
 
+2. Verify secret exists (the result is encrypted).
+```shell
+kubectl get secrets -n langflow openai-credentials
+```
 
-### Scaling {#359b9ea5302147ebbed3ab8aa49dae8d}
+3. Upgrade the helm release to use the secret.
+```shell
+helm upgrade my-langflow-app-image langflow/langflow-runtime -n langflow \
+  --reuse-values \
+  --set "extraEnv[0].name=OPENAI_API_KEY" \
+  --set "extraEnv[0].valueFrom.secretKeyRef.name=openai-credentials" \
+  --set "extraEnv[0].valueFrom.secretKeyRef.key=OPENAI_API_KEY"
+  ```
 
+### Configure storage
 
-You can scale the number of replicas for the LangFlow app by changing the `replicaCount` value in the `values.yaml` file.
+In this case, storage is not needed, as your deployment is stateless.
```

Comment:
Not sure why we have a section to configure storage that tells people that they don't have to configure storage.